src/big_widgets/GitQlient.cpp

```diff
@@ -112,8 +106,7 @@ GitQlient::GitQlient(const QStringList &arguments, QWidget *parent)
 
    connect(mConfigWidget, &InitScreen::signalOpenRepo, this, &GitQlient::addRepoTab);
 
-   setRepositories(repos);
-
+   GitQlientSettings settings;
```

Comment:
This is my bad during the merge. Can you remove it it as part of the other fixes?

Reply:
Sure